release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -772,6 +772,8 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-12-bare-metal-hardware-bug-fixes"]
 ==== Bare Metal Hardware Provisioning
 
+* Before this update, when a Redfish system features a Settings URI, the Ironic provisioning service always attempts to use this URI to make changes to boot related BIOS settings. However, bare-metal provisioning fails if the Baseboard Management Controller (BMC) features a Settings URI but does not support changing a particular BIOS setting by using this Settings URI. In {product-title} 4.12 and later, if a system features a Settings URI, Ironic verifies that it can change a particular BIOS setting by using the Settings URI before proceeding. Otherwise, Ironic implements the change by using the System URI. This additional logic ensures that Ironic can apply boot-related BIOS setting changes and bare-metal provisioning can succeed. (link:https://issues.redhat.com/browse/OCPBUGS-2052[*OCPBUGS-2052*])
```

Comment:
```suggestion
* Previously, the Ironic provisioning service on a Redfish system always attempted to use Settings URI to make changes to boot related BIOS settings. However, bare-metal provisioning fails if the Baseboard Management Controller (BMC) features a Settings URI but does not support changing a particular BIOS setting by using the Settings URI. In {product-title} 4.12 and later, if a system features a Settings URI, Ironic verifies that it can change a particular BIOS setting by using the Settings URI before proceeding. Otherwise, Ironic implements the change by using the System URI. This additional logic ensures that Ironic can apply boot-related BIOS setting changes and bare-metal provisioning can succeed. (link:https://issues.redhat.com/browse/OCPBUGS-2052[*OCPBUGS-2052*])
```

Reply:
Thanks Joe, I think the Settings URI is not always present, hence the slightly clunky phrasing. For that reason I might keep this as is. Also, I think the guidance for bug fixes is to use "Before this update" rather than previously - although I don't know how hard a rule this is: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

Appreciate the feedback. :-)